### PR TITLE
Restructure Website documentation

### PR DESCRIPTION
**Pull request by svrnm**
This PR restructures the website documentation in a way that it is closer to what other languages have (dedicated pages for exporters, resources, sampling), cc @open-telemetry/docs-approvers

### CODE REVIEW

File: website_docs/getting-started.md [hunk 1]
```diff
@@ -542,10 +542,12 @@ start adding OpenTelemetry Go to your projects at this point. Go instrument your
...
-spans, refer to the [Instrumenting]({{< relref "manual" >}})
-documentation. Likewise, advanced topics about processing and exporting
-telemetry data can be found in the [Processing and Exporting Data]({{< relref
-"exporting_data" >}}) documentation.
+spans, refer to the [manual instrumentation](/docs/instrumentation/go/manual/)
+documentation.
+
+You'll also want to configure an appropriate exporter to [export your telemetry
+data](/docs/instrumentation/go/exporters) to one or more telemetry backends.
+
```
Comment (hanyuancheung) [inline website_docs/getting-started.md:550]:
```suggestion
```
The deprecated blank line should be removed.

**Review by hanyuancheung — APPROVED** [review]

File: website_docs/exporters.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+---
+title: Exporters
+weight: 4
+---
```
Comment (chalin) [inline website_docs/exporters.md:4]: Add an alias entry for `exporting_data`.

**Review by chalin — COMMENTED** [review]
See inline comment.

File: website_docs/exporters.md (hunk 2 above)
Comment (svrnm) [reply website_docs/exporters.md:5]: done

File: website_docs/exporters.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+title: Exporters
+aliases: [/docs/instrumentation/go/exporting_data/]
```
Comment (chalin) [inline website_docs/exporters.md:3]:
```suggestion
aliases: [/docs/instrumentation/go/exporting_data]
```

**Review by chalin — APPROVED** [review]
See inline comment for a nit fix.

Is there anything holding back a final review and merge of this PR?

**Comment by svrnm** [comment]
@open-telemetry/go-approvers PTAL

File: website_docs/exporters.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+OpenTelemetry Protocol (OTLP) export is available in the
+`go.opentelemetry.io/otel/exporters/otlp/otlptrace` and
+`go.opentelemetry.io/otel/exporters/otlp/otlpmetric` packages.
+
+Please find more documentation on
+[GitHub](https://github.com/open-telemetry/opentelemetry-go/tree/main/exporters/otlp)
+
+## Jaeger Exporter
+
+Jaeger export is available in the `go.opentelemetry.io/otel/exporters/jaeger`
```
Comment (dmathieu) [inline website_docs/exporters.md:22]: How about including the content of #3705 here, to avoid losing data between the two PRs?

Comment (svrnm) [reply website_docs/exporters.md:22]: would be good, since this PR is outstanding for a month now 👍

**Review by dmathieu — APPROVED** [review]

**Comment by chalin** [comment]
Woohoo, thanks all!

**Pull request merged**